src/main/java/bio/terra/service/snapshot/Snapshot.java

```diff
@@ -180,4 +182,13 @@ public AzureRegion getStorageAccountRegion() {
   public boolean isSecureMonitoringEnabled() {
     return getFirstSnapshotSource().getDataset().isSecureMonitoringEnabled();
   }
+
+  public SnapshotRequestContentsModel getMode() {
```

Comment:
I'm not sure mode is the right name for this since this includes more than just the mode.  I could see something like `spec`, `specification`, or `creationInfo`...or something along those lines